src/components/widgets/thermals/ThermalChart.vue

```diff
@@ -292,7 +292,7 @@ export default class ThermalChart extends Vue {
 
   createSeries (label: string, key: string) {
     // Grab the color
-    const color = Vue.$colorset.next(getKlipperType(key), key)
```

Comment:
This is an unrelated change, but the TS compiler wouldn't let me continue without fixing it